content/docs/SUSHI/running/_index.md

```diff
@@ -91,27 +97,19 @@ customized-ig
 │   │   ├── mySecondPage.md
 │   │   ├── myThirdPage.md
 │   │   └── myFourthPage.md
-│   ├── profiles
-│   │   └── StructureDefinition-myProfile.json
-│   └── vocabulary
-│       ├── ValueSet-myValueSet.json
-│       └── CodeSystem-myCodeSystem.json
-└── package-list.json
+├── package-list.json
+└── sushi-config.json
 ```
 
-{{% alert title="Info" color="info" %}}
-SUSHI generates _output_ into a directory called **input**, because it is generating _input_ files for the IG Publisher.  The IG Publisher requires this directory to be named **input**.
-{{% /alert %}}
-
 Note the following files and directories from the output:
 
-* **ig.ini**: Copied from the **fsh/ig-data** directory, but can alternately be specified via the `template` property in **fsh/config.yaml**.
-* **input/ignoreWarnings.txt**: Copied from the **fsh/ig-data/input** directory, but will be generated as a blank file if it isn't found.
-* **input/examples\***, **input/extensions\***, **input/profiles\***, **input/vocabulary\***: Generated from the definitions in the **fsh/\*.fsh** files.
-* **input/images/\***: Copied from the **fsh/ig-data/input/images** directory.
-* **input/includes/menu.xml**: Copied from the **fsh/ig-data/input/includes** directory, but can alternately be specified via the `menu` property in **fsh/config.yaml**.
-* **input/pagecontent/\***: Copied from the **fsh/ig-data/input/pagecontent** directory. Filenames with numeric prefixes will have the prefixes removed.
-* **package-list.json**: Copied from the **fsh/ig-data** directory, but can alternately be specified via the `history` property in **fsh/config.yaml**.
+* **ig.ini**: Specified by the author and unchanged by SUSHI.
+* **input/ignoreWarnings.txt**: Specified by the author and unchanged by SUSHI.
+* **fsh-generated\***: Generated from the definitions in the **input/fsh/\*.fsh** files.
```

Comment:
I think I'd suggest moving this `fsh-generated` bullet point to the top of the list just to follow the alphabetic order in the output above.